src/middlewares/middlewares.ts

```diff
@@ -1,4 +1,33 @@
 import { NextFunction, Request, Response } from 'express';
+import { Strategy, ExtractJwt } from 'passport-jwt';
+import passport from 'passport';
+import User from '../models/user';
```

Comment:
Same import issue as before. Let's create an index file in the models folder and export everything from there. After that let's change the import
```suggestion
import passport from 'passport';
import { User }  from '../models';
```